scipy/stats/__init__.py

```diff
@@ -456,6 +456,14 @@
 The following functions are related to the tests above but do not belong in the
 above categories.
 
+Random Variables
```

Comment:
should we rather say "Continuous distributions (new infrastructure)"? "Random Variables" does not really describe the functionality in my view

Reply:
I think it's fair to call an instance of `StandardNormal` a random *variable*, but not a random *variate*. And I guess that that would make the `StandardNormal` *type* the analogue of the standard normal *distribution* then. 

So it a matter of preference I guess in this case 🤷🏻.

The only that I can conclude with this, is that the "legacy" `rv_generic` shouldn't have been called an "RV". 

So hopefully that'll soon be replaced with a better version 😉